src/globals.d.ts

```diff
@@ -0,0 +1 @@
+declare const __PACKAGE_VERSION__: string;
```

Comment:
Linter fails here. It requires a new line to be added at the end of the file.
```suggestion
declare const __PACKAGE_VERSION__: string;

```